cypress/e2e/features/Ui/doubleVanities.feature

```diff
@@ -10,3 +10,22 @@ Given User navigates to Home Page
     Scenario: Test All Links on the Double Vanities Page
     Given User clicks to Double Bathroom Vanities
     When I should see all links are working
+
+    Scenario Outline: Perform a search with different terms
+    Given User clicks to Double Bathroom Vanities
+    When I enter "<searchTerm>" into the search box
+    Then I should see "<expectedResultCount>" results for "<searchTerm>"
+
+    Examples:
+      |searchTerm         |expectedResultCount|    
+      |White Selena Double| 3                 |
+      |Navy Blue Selena   | 3                  |
```

Comment:
What will happen if they add one more Navy Blue Selena to the page? If you set the number of products as expected test data, you need to keep changing your test. Which will be very hard to maintain